spaces/S000070/properties/P000166.md

```diff
@@ -1,10 +1,11 @@
 ---
 space: S000070
-property: P000009
+property: P000166
 value: true
 refs:
 - doi: 10.1007/978-1-4612-6290-9_6
   name: Counterexamples in Topology
 ---
+The topology is finer than that of $\mathbb R\times[0,\infty)$ as a subspace of {S176}.
 
 See item #2 for space #78 in {{doi:10.1007/978-1-4612-6290-9_6}}.
```

Comment:
Is this still a relevant section of *Counterexamples*?

Reply:
(See also other edited traits)

Reply:
Yes. I made sure that the references were still applicable.